client/me/purchases/controller.jsx

```diff
@@ -19,7 +19,7 @@ import EditCardDetails from './payment/edit-card-details';
 import Main from 'components/main';
 import ManagePurchase from './manage-purchase';
 import NoSitesMessage from 'components/empty-content/no-sites-message';
-import paths from './paths';
+import * as paths from './paths';
```

Comment:
It seems `billingHistory` and `billingHistoryReceipt` are not used but are still imported this way. Do we want to specify all the imports instead of just importing them all?

Reply:
I felt like since we're using almost all of it that it was warranted to use the `*` import but if you feel strongly about this we can change it.

On this note it's on my TODO list (raised by @samouri already) to check if the `*` import actually imports all of the exports or only the ones statically used (once CommonJS is gone).

Reply:
Ah, it's totally up to you (I don't feel strongly about it), just saying I'd probably list all the imports rather than `*` - `almostAll !== all` 😉

Reply:
turns out `import *` does get tree shaken.  In general I think its cleaner to use `import *` when number of imports > 3.  Less agonizing over alphabetic sorting and deleting/adding imports

source: https://github.com/webpack/webpack/issues/2713

Reply:
TIL 💡 👍